word2gauss/embeddings.pyx

```diff
@@ -569,7 +569,7 @@ cdef class GaussianEmbedding:
                 for p in ph_tok:
                     if isinstance(p, basestring):
                         phrase_vec += self.mu[vocab.word2id(p), :]
-                phrase_vec /= len(ph_tok)
+                if len(ph_tok) != 0: phrase_vec /= len(ph_tok)
```

Comment:
I would prefer this to use normal Python style of two lines for the `if` statement rather than squeezing this onto a single line, but this change does look reasonable.